src/resolve/resolved-dns-browse-services.c

```diff
@@ -0,0 +1,776 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "resolved-dns-browse-services.h"
+#include "af-list.h"
+#include "event-util.h"
+#include "json-util.h"
+#include "random-util.h"
+#include "resolved-dns-cache.h"
+#include "resolved-varlink.h"
+#include "string-table.h"
+
+typedef enum BrowseServiceUpdateFlag {
```

Comment:
flags are bit fields for us, this one is not, hence maybe call this BrowseEvent or so